ChickenButt.jsx

```diff
@@ -0,0 +1,6 @@
+module.exports = (msg, client) => {
+    const lowerCase = msg.content.toLowerCase();
+    if (lowerCase.startsWith("What ") || lowerCase.startsWith("what ")) {
```

Comment:
msg.content.toLowerCase() is never going to start with "What ", so this if statement can be written to only check if lowerCase starts with "what "